### PR TITLE
Add monocle version changes and fix tests

### PR DESCRIPTION
**Pull request by kshitiz-okahu**
## Proposed changes

Added monocle version in span
Fix tests

## Types of changes

What types of changes does your code introduce to this project?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/monocle2ai/monocle/blob/main/CONTRIBUTING.md) doc
- [ ] I have signed the CLA
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added the necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by kshitiz-okahu** [comment]
add span.type in workflow span

**Review by oi-raanne — APPROVED** [review]

File: .github/workflows/tox.yml [hunk 1]
```diff
@@ -4,12 +4,10 @@ name: Tox
 
 on:
   push:
-    branches: [ "main" ]
+    branches: [ "main", "kshitiz/update_test" ]
```
Comment (oi-raanne) [inline .github/workflows/tox.yml:7]: pls change this

**Review by oi-raanne — CHANGES_REQUESTED** [review]

File: .github/workflows/tox.yml (hunk 1 above)
Comment (kshitiz-okahu) [reply .github/workflows/tox.yml:7]: yes, just fixing the last failing test

**Review by oi-raanne — APPROVED** [review]

File: src/monocle_apptrace/wrap_common.py [hunk 2]
```diff
@@ -163,6 +164,14 @@ def post_task_processing(to_wrap, span, return_value):
 def pre_task_processing(to_wrap, instance, args, span):
     try:
         if is_root_span(span):
+            workflow_name = span.resource.attributes.get("service.name")
+            span.set_attribute("workflow_name", workflow_name)
+            
+            try:
+                sdk_version = version("monocle_apptrace")
+                span.set_attribute("monocle_sdk_version",sdk_version)
```
Comment (oi-raanne) [inline src/monocle_apptrace/wrap_common.py:172]: can the attribute be `monocle_apptrace.version` to reflect the package name

**Review by oi-raanne — CHANGES_REQUESTED** [review]

**Pull request closed without merging**